.github/workflows/attach-artifact-release.yml

```diff
@@ -14,85 +14,99 @@ on:
-              with:
-                  name: liquibase-couchbase
+            -   name: Publish Artifacts
+                uses: actions/upload-artifact@v2
+                with:
+                    name: liquibase-couchbase
+                    path: liquibase-couchbase/target/*
 
-            - name: Get Release Tag
-              id: get-release-tag
-              run: echo "release_tag=$(./.github/get_draft_release.sh TAG)" >> $GITHUB_ENV
-              env:
-                  GITHUB_TOKEN: ${{ secrets.BOT_TOKEN }}
+            -   name: Get Reusable Script Files
+                run: |
+                    curl -o $PWD/.github/get_draft_release.sh https://raw.githubusercontent.com/liquibase/build-logic/v0.3.1/.github/get_draft_release.sh
+                    curl -o $PWD/.github/sign_artifact.sh https://raw.githubusercontent.com/liquibase/build-logic/v0.3.1/.github/sign_artifact.sh
+                    curl -o $PWD/.github/upload_asset.sh https://raw.githubusercontent.com/liquibase/build-logic/v0.3.1/.github/upload_asset.sh
+                    chmod +x $PWD/.github/get_draft_release.sh
+                    chmod +x $PWD/.github/sign_artifact.sh
+                    chmod +x $PWD/.github/upload_asset.sh
 
-            - name: Delete Outdated Files from Draft Release
-              id: delete-outdated-release-asset
-              uses: mknejp/delete-release-assets@v1
-              with:
-                  token: ${{ secrets.BOT_TOKEN }}
-                  tag: ${{ env.release_tag }}
-                  fail-if-no-assets: false
-                  fail-if-no-release: false
-                  assets: "${{ env.artifact_id }}-*"
+            -   name: Download Artifacts
+                uses: actions/download-artifact@v2
+                with:
+                    name: liquibase-couchbase
 
-            - name: Import GPG key
-              id: import_gpg
-              uses: crazy-max/ghaction-import-gpg@v5
-              with:
-                  gpg_private_key: ${{ secrets.GPG_SECRET }}
-                  passphrase: ${{ secrets.GPG_PASSPHRASE }}
+            -   name: Get Release Tag
+                id: get-release-tag
+                run: echo "release_tag=$(./.github/get_draft_release.sh TAG)" >> $GITHUB_ENV
+                env:
+                    GITHUB_TOKEN: ${{ secrets.BOT_TOKEN }}
 
-            - name: Sign Files for Draft Release
-              run: |
-                  gpg -K
-                  version=$(mvn help:evaluate -Dexpression=project.version -q -DforceStdout)
-                  ./.github/sign_artifact.sh ./assets/${{ env.artifact_id }}-${version}.jar
-                  ./.github/sign_artifact.sh ./assets/${{ env.artifact_id }}-${version}.pom
-                  ./.github/sign_artifact.sh ./assets/${{ env.artifact_id }}-${version}-javadoc.jar
-                  ./.github/sign_artifact.sh ./assets/${{ env.artifact_id }}-${version}-sources.jar
+            -   name: Delete Outdated Files from Draft Release
```

Comment:
What's the source of extra spaces here between `-` and text? Is it a non-formatted old code or a different codestyle for yml files (or lack of formatting configuration for yml files)?

Reply:
A very good question. I use default with .editorconfig and it somehow overrides default indent of 2 symbols.